src/data/markdown/docs/02 javascript api/07 k6-experimental/06 webcrypto/02 SubtleCrypto/06 importKey.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'importKey'
+excerpt: 'importKey imports a key from an external, portable format and gives you a CryptoKey object.'
+---
+
+The `importKey()` imports a key from an external, portable format, and gives you a [CryptoKey](/javascript-api/k6-experimental/webcrypto/cryptokey) object that can be used with the Web Crypto API.
+
+## Usage
+
+```
+importKey(format, keyData, algorithm, extractable, keyUsages)
+```
+
+## Parameters
+
+| Name          | Type                                                      | Description                                                                                                                                                   |
+| :------------ | :-------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `format`      | `string`                                                  | Defines the data format of the key to import. Currently supported formats: `raw`.                                                                             |
+| `keyData`     | `ArrayBuffer`, `TypedArray` or `DataView`                 | the data to import the key from.                                                                                                                              |
+| `algorithm`   | a `string` or object with a single `name` string property | The algorithm to use to import the key. Currently supported algorithms: `AES-CBC`, `AES-GCM`, `AES-CTR`, and `HMAC`.                                          |
+| `extractable` | `boolean`                                                 | Indicates whether it will be possible to export the key using [exportKey](/javascript-api/k6-experimental/webcrypto/subtlecrypto/exportkey).                  |
+| `keyUsages`   | `Array<string>`                                           | An array of strings describing what operations can be performed with the key. Currently supported usages include: `encrypt`, `decrypt`, `sign`, and `verify`. |
```

Comment:
```suggestion
| `keyUsages`   | `Array<string>`                                           | An array of strings describing what operations can be performed with the key. Currently supported usages include `encrypt`, `decrypt`, `sign`, and `verify`. |
```

Reply:
In these cases the colon is unnecessary because the list is the direct object of "Include". We do use colons to introduce items in HTML lists, but that's "incorrect" in a certain stylistic sense. I guess the colon makes the sentence look complete, which isn't a problem when the list is embedded in the same line.

Reply:
Fair enough 🙇‍♂️